packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-julia.vue

```diff
@@ -205,17 +250,31 @@ const selectedRunId = computed(
 		Object.values(props.node.state.calibrateConfigs.runConfigs).find((metadata) => metadata.active)
 			?.runId
 );
+
 const drilldownLossPlot = ref<HTMLElement>();
+const staticLossPlotRef = ref<HTMLElement>();
+let lossValues: { [key: string]: number }[] = [];
 
-// Tom TODO: Make this generic... its copy paste from node.
-const chartConfigurationChange = (index: number, config: ChartConfig) => {
-	const state = _.cloneDeep(props.node.state);
-	state.calibrateConfigs.chartConfigs[index] = config.selectedVariable;
+// refs to keep track of intermediate states and parameters
+const currentIntermediateVals = ref<{ [key: string]: any }>({ timesteps: [], solData: {} });
+const parameterResult = ref<{ [index: string]: any }>();
 
-	emit('update-state', state);
-};
+const showSpinner = ref(false); // TODO: showSpinner isn't actually showing a spinner -> rename this or maybe show a spinner
```

Comment:
This todo looks out of date, show spinner is being used above?

Reply:
just got rid of it